packages/jellyfish-api-core/__tests__/category/account.test.ts

```diff
@@ -516,4 +516,81 @@ describe('masternode', () => {
       expect(data.length).toBe(64)
     })
   })
+
+  describe('accountHistoryCount', () => {
+    it('should get accountHistoryCount', async () => {
+      await waitForExpect(async () => {
+        const count = await client.account.historyCount()
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThanOrEqual(0)
+      })
+    })
+
+    it('should get accountHistoryCount with owner as all', async () => {
+      await waitForExpect(async () => {
+        const count = await client.account.historyCount('all')
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThanOrEqual(0)
+      })
+    })
+
+    it('should get accountHistoryCount with no_rewards option', async () => {
+      await waitForExpect(async () => {
+        const options: AccountHistoryCountOptions = {
+          no_rewards: true
+        }
+        const count = await client.account.historyCount('mine', options)
+
+        expect(typeof count).toBe('number')
+        expect(count).toBeGreaterThanOrEqual(0)
+      })
+    })
+
+    it('should get accountHistoryCount with token option', async () => {
+      await waitForExpect(async () => {
+        const options1: AccountHistoryCountOptions = {
+          token: 'DBTC'
+        }
+        const options2: AccountHistoryCountOptions = {
+          token: 'DETH'
+        }
+        const countWithDBTC = await client.account.historyCount('mine', options1)
+        const countWithDETH = await client.account.historyCount('mine', options2)
+
+        expect(typeof countWithDBTC).toBe('number')
+        expect(typeof countWithDETH).toBe('number')
+        expect(countWithDETH === countWithDBTC).toStrictEqual(false)
```

Comment:
hmm... this is not my expect actually..
my previous request was actually requesting a meaningful test.. you should ady known the `count` as you can refer to the fixture set up above, so the test should be telling a story how the `accounthistorycount` work
you can straight away assert the exact number since its fixed

Reply:
@canonbrother , you mean something like `expect(count).toStrictEqual(10)` ?

Reply:
yea right